packages/cfd/src/Components/__tests__/cfd-poi.spec.js

```diff
@@ -130,30 +130,6 @@ describe('<CFDPOI />', () => {
         expect(props.onCancel).toHaveBeenCalledTimes(1);
     });
 
-    it('should enable Next button when identity_status is pending or verified, and disable otherwise', () => {
```

Comment:
This test case is no longer valid. When users submit manual upload, we do not want to make an unnecessary API call to get a status update, but we want to enable the button to allow users to proceed to POA submit.